Code/GraphMol/RGroupDecomposition/RGroupScore.h

```diff
@@ -13,6 +13,7 @@
 #include "RGroupMatch.h"
 #include <vector>
 #include <set>
+#include <boost/serialization/vector.hpp>
```

Comment:
It's not clear why this is here.
If it's necessary, it should at least be behind `#ifdef RDK_USE_BOOST_SERIALIZATION`

Reply:
I've removed it.  Not sure how it got there- my guess is I selected the wrong vector during autocomplete.